nav2_behavior_tree/plugins/action/remove_passed_goals_action.cpp

```diff
@@ -29,22 +29,33 @@ RemovePassedGoals::RemovePassedGoals(
   const std::string & name,
   const BT::NodeConfiguration & conf)
 : BT::ActionNodeBase(name, conf),
-  viapoint_achieved_radius_(0.5)
+  viapoint_achieved_radius_(0.5),
+  initialized_(false)
+{}
+
+void RemovePassedGoals::initialize()
 {
   getInput("radius", viapoint_achieved_radius_);
 
   tf_ = config().blackboard->get<std::shared_ptr<tf2_ros::Buffer>>("tf_buffer");
   auto node = config().blackboard->get<rclcpp::Node::SharedPtr>("node");
   node->get_parameter("transform_tolerance", transform_tolerance_);
 
+  global_frame_ = BT::deconflictPortAndParamFrame<std::string, RemovePassedGoals>(
```

Comment:
I believe this was removed. Maybe you're working on an old commit?

Reply:
fixed